app/views/shared/search/_keyword.html.slim

```diff
@@ -1,4 +1,11 @@
-= f.govuk_text_field :keyword,
-  label: { text: t("jobs.filters.keyword"), size: "s" },
-  placeholder: t("jobs.filters.keyword_hint"),
-  width: filters ? "full" : "two-thirds"
+- if @origin == "/sign-up-for-NQT-job-alerts"
```

Comment:
This string is duplicated in several places so I'd like to find a way to get it programmatically.

Reply:
Is it possible to use fewer lines here by putting the `if` inside the hint text? Not sure what happens if you pass `nil` or `""` to `hint: { text: }`.

```ruby
hint: { text: (@origin == "/sign-up-for-NQT-job-alerts" ? t("jobs.filters.keyword_nqt_hint") : nil },
```